package.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "phaser-grunt-boilerplate",
+  "name": "craft",
```

Comment:
do we want this to be `@code-dot-org/craft` to match our other npm-installed projects?